src/Conferences/ConferencesHarvester.php

```diff
@@ -117,7 +118,8 @@ private function retrieveConferenceIfExists(Conference $conference): ?Conference
                     $existingConference['startAt']->format('Y-m-d') === $conference->getStartAt()->format('Y-m-d')
                     && $existingConference['endAt']->format('Y-m-d') === $conference->getEndAt()->format('Y-m-d')
                 ) {
-                    $matchedConference = $conference;
+                    $existingConference = $this->conferenceRepository->findOneBy(['id' => $existingConference['id']]);
+                    $matchedConference = $existingConference;
```

Comment:
There is not break here, so we could fetch the "real" conference as late as possible to avoid extra SQL query

Reply:
Fixed in https://github.com/jolicode/starfleet/pull/133